Source/WebCore/dom/DocumentMarker.h

```diff
@@ -118,17 +118,16 @@ class DocumentMarker : public CanMakeWeakPtr<DocumentMarker> {
 #endif
 
 #if ENABLE(WRITING_TOOLS)
-    struct UnifiedTextReplacementData {
+    struct WritingToolsTextSuggestionData {
         enum class State: uint8_t {
-            Pending,
-            Committed,
-            Reverted
+            Accepted,
+            Rejected
         };
 
         String originalText;
-        WritingTools::TextSuggestionID replacementID;
+        WritingTools::TextSuggestionID suggestionID;
         WritingTools::SessionID sessionID;
-        State state { State::Pending };
+        State state { State::Accepted };
```

Comment:
Just checking this is correct? (just seems wrong, but know the naming was weird)

Reply:
Ya; it doesn't really matter what the default is anyways since it's always initialized, but by default it's accepted anyways too